metric/doc.go

```diff
@@ -30,6 +30,59 @@ can then be built from the [Meter]'s instruments. See
 [go.opentelemetry.io/otel/metric/instrument] for documentation on each
 instrument and its intended use.
 
+# API Implementations
+
+This package does not conform to the standard Go versioning policy, all of its
+interfaces may have methods added to them without a package major version bump.
+So an implementation author is not surprised by this fact, the API is designed
+to require a default behavior be defined for unimplemented interface methods.
+There are three behavior choices:
```

Comment:
The existing sentence builds from the context of the previous sentence "This package does not conform to the standard Go versioning policy, all of its interfaces may have methods added to them without a package major version bump."

The proposed change will introduce a jump in sentence flow. I'm not sure it is appropriate.

Reply:
1.  Then maybe just changing `So` to `Therefore,`?
2.  Can split it somehow into 2 sentences? I find the sentence too complex (contains many verbs).
3. I am not sure if `an implementation author is not surprised by this fact` adds any value (or maybe it could be phrased better). 


```suggestion
Therefore, an implementation author is not surprised, as the API is designed
to require a default behavior be defined for unimplemented interface methods.
There are three behavior choices:
```

PS. I wish I were better in English 😅

Reply:
I've reworked this section and pushed a change. Please take another look.

Reply:
I really like the changes! Good job.